libs/stokesian_dynamics/sd_cpu.hpp

```diff
@@ -1,4 +1,5 @@
-#pragma once
```

Comment:
This is a bit ridiculous.  I can't imagine that there is any compiler where Espresso compiles that does not support `#pragma once`.

Reply:
It's actually not for compiler support reason, but for CMake safety. We have a few header files that CMake copies around, and making espresso a traditional include guard-only codebase avoids any risk of accidentally including the same header twice because of a different filepath. However in included libraries, I guess `#pragma once` should be fine?

Reply:
Let's treat is as a library with its own code style guidelines. We don't run clang-tidy over the libs directory either.